R/MsBackendTimsTof.R

```diff
@@ -74,7 +86,11 @@ setClass("MsBackendTimsTof",
                    indices = "matrix",
                    fileNames = "character"),
          prototype = prototype(frames = data.frame(),
-                               indices = matrix(),
+                               indices = matrix(, nrow = 0, ncol = 3,
```

Comment:
is the `,` in `matrix(, nrow = ...)` here intentional?

Reply:
I thought it was necessay but actually it isn't. I'll remove it! Thanks!